website/docs/guides/migration/versions/01-upgrading-to-v1.6.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Upgrading to v1.6 (beta)"
+description: New features and changes in dbt Core v1.6
+---
+
+:::warning Beta Functionality
```

Comment:
Should we make this note a Snippet to reuse on all pages affected? 

Also (This might be something for a future front-end wish list cc @JKarlavige) it would be cool if the note automatically showed up on all affected pages.  I mean, we already know which version is a prerelease because we specify that [here](https://github.com/dbt-labs/docs.getdbt.com/pull/3353/files#diff-2c10d5fd88d9179903ad35f517bbb370092dcfe4cc5730bf5a04413a3860e9a8R5) and we know which pages should be versioned because we add them to the versioning page. It'd be nice to automate things like this.